src/tram/templates/index.html

```diff
@@ -72,9 +72,9 @@ <h1>Reports</h1>
             </button>
             <div class="dropdown-menu" aria-labelledby="export-dropdown">
               <a class="dropdown-item btn btn-sm btn-outline-secondary"
-                href="/api/report-export/{{ report.id }}/?type=json">JSON</a>
+                href="/api/reports/{{ report.id }}/json">JSON</a>
```

Comment:
Hardcoded URLs should be avoided when possible:
```suggestion
                href="{% url 'report-json' report.id %}">JSON</a>
```